packages/pigment-css-react/tests/styled/fixtures/styled-theme.input.js

```diff
@@ -34,3 +35,18 @@ const SliderRail2 = styled.span`
     display: none;
   }
 `;
+
+export function App() {
+  return (
+    <Component>
+      <SliderRail />
+      <SliderRail2 />
+    </Component>
+  );
+}
+
+process.env.NODE_ENV !== 'production'
+  ? (App.propTypes = {
+      children: PropTypes.element,
+    })
+  : void 0;
```

Comment:
What would happen if the propTypes are not handled? I guess the test will throw during evaluation?

Reply:
Correct.

Reply:
During evaluation, App gets set to null. So we are setting a value to null object resulting in the error.

```
null.propTypes = {}
```